src/Application/Redux/StateUtils.js

```diff
@@ -0,0 +1,157 @@
+import ImmutableAsset from './../Models/Asset';
+import { List } from 'immutable';
+import _ from 'underscore';
+
+const PRICE_INCREASE_PERCENTAGE = 7;
+const BASE_CLICK_INCOME = 1;
+
+const createNews = (news, newArticle) => {
+  const copyOfNews = news.slice();
+  copyOfNews.push(newArticle);
+  if (copyOfNews.length > 10) copyOfNews.shift();
+  return copyOfNews;
+};
+
+const unlockAsset = (asset) => {
+  return Object.assign({},
+    asset,
+    {
+      unlocked: true,
+    }
+  );
```

Comment:
also we actually tend to just use the constructor on the model class instead of `Object.assign`.  You have to write out all the arguments but it's more explicit.  might not be any difference here tho